python/feathub/common/utils.py

```diff
@@ -88,6 +88,24 @@ def append_unix_time_column(
     return unix_time_column
 
 
+def append_epoch_millis_column(
```

Comment:
Can we unify this method with the existing method `append_unix_time_column`?

It seems a bit weird to put this method (and also `append_unix_time_column`) in this file given that this method is expected to be used only by LocalProcessor. Can you help refactor it similar to the file pattern used by FlinkProcessor and SparkProcessor?